.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**PDF files**
+Check if the bug appear only with some PDF. If the bug is specific to some PDF files please attach one to the issue.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System and Versions**
+ - PDF-Arranger version and where you get it (distro package, flatpak, pip, git, download, ...)
```

Comment:
I'd prefer "PDF Arranger" to be consistent. "got" could be used as past tense of "get" but the message is clear anyway.